example/evaluate_policy.py

```diff
@@ -104,9 +104,9 @@ def main():
 
     else:
         env = gym.make(
-            "trifinger_simulation.gym_wrapper:real_robot_challenge_phase_1-v1",
+            "TriFingerCubeDifficulty{}-v1".format(difficulty),
```

Comment:
Does this working without specifying the package?

Reply:
yep it does :)

Reply:
Okay, in this case I guess you can ignore my comment regarding the names above.

Reply:
I am actually looking at this in more detail now.. I will get back to you after that